content/4_Gold/TopoSort.mdx

```diff
@@ -382,13 +383,13 @@ void compute() {
 <JavaSection>
 
 ```java
-static int in_degree[];
-static ArrayList<Integer> edge[]; //adjacency list
+static int[] in_degree;
```

Comment:
camelCase? (there's some other places that don't have it, be sure to do that)
also uh wrap this in a class